<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Convert</class>
 <widget class="QWidget" name="Convert">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>429</width>
     <height>89</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>ASCII:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtACSII"/>
      </item>
      <item>
       <widget class="QPushButton" name="btn1">
        <property name="text">
         <string>转换</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>HEX:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtHex1"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>HEX:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtHex2"/>
      </item>
      <item>
       <widget class="QPushButton" name="btn2">
        <property name="text">
         <string>转换</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>ASCII:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtASCI2"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>字符：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtChar"/>
      </item>
      <item>
       <widget class="QPushButton" name="btn3">
        <property name="text">
         <string>转换</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>HEX:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtHex3"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
